src/fast_check/transform.rs

```diff
@@ -442,6 +509,11 @@ impl<'a> FastCheckTransformer<'a> {
         if !self.public_ranges.contains(&public_range) {
           return Ok(false);
         }
+
+        self
+          .public_inferred_namespaces
+          .insert(n.ident.sym.to_string(), vec![]);
```

Comment:
Instead of a string, maybe let's store `n.ident.sym` in the hashmap because that takes scope into account too. Here's a test case that shouldn't have a `skip` property:

```ts
export function test() {
}

if (true) {
  function test() {}
  test.skip = () => {};
}
```